_studio/mfx_lib/encode_hw/hevc/agnostic/base/hevcehw_base_legacy.cpp

```diff
@@ -3568,7 +3568,7 @@ void Legacy::SetDefaults(
     };
     auto GetDefaultLevel = [&]()
     {
-        mfxU16 nCol, nRow;
+        mfxU16 nCol = 0, nRow = 0;
```

Comment:
Since these are initialized by the std::tie function, I don't think you need to explicitly initialize them, but you can.  I would call this a KW false positive.